app/workers/kyc/kycaid/address_worker.rb

```diff
@@ -0,0 +1,50 @@
+require 'sidekiq'
+
+module KYC
+  module Kycaid
+    class AddressWorker
+      include Sidekiq::Worker
+
+      def perform(params = {})
+        @params = params.symbolize_keys
+        
+        @user = User.find(@params[:user_id])
+        @applicant_id = @user.profiles.last.applicant_id
+        @document = Document.find_by(identificator: @params[:identificator])
+        address = ::KYCAID::Address.create(address_params)
+
+        if address.error || address.errors
+          Rails.logger.info("Error in document creation for: #{@user.uid}: #{address.errors} #{address.error}")
+          @user.labels.find_by(key: :address, scope: :private).update(value: 'rejected')
+        elsif address.document_id
```

Comment:
miscommunication. On Address on gem level we create a Document (with a document_id) but in the end we recieve response from Address generation (linked with doc), so truly, @mnaichuk is right, should be address_id
https://docs.kycaid.com/#create-an-address
https://github.com/openware/kycaid-ruby/blob/master/lib/kycaid/address.rb#L27